src/OpenIddict.Client/OpenIddictClientService.cs

```diff
@@ -265,9 +265,11 @@ public async ValueTask<InteractiveAuthenticationResult> AuthenticateInteractivel
                     AuthorizationCode = context.AuthorizationCode,
                     AuthorizationResponse = context.Request is not null ? new(context.Request.GetParameters()) : new(),
                     BackchannelAccessToken = context.BackchannelAccessToken,
+                    BackchannelAccessTokenExpirationDate = context.BackchannelAccessTokenExpirationDate!,
```

Comment:
Is there a particular reason for using the null forgiving operator? AFAICT, all the properties you added in this PR are nullable?

Reply:
None whatsoever, pushed fix